### PR TITLE
Bitcoin support

### PR DESCRIPTION
**Pull request by vevix**
:moneybag: :moneybag: :moneybag: :moneybag: :moneybag: :moneybag: :moneybag: :moneybag: :moneybag: :moneybag:

### CODE REVIEW

File: app/frontend/app/controllers/pro.js [hunk 1]
```diff
@@ -104,7 +103,9 @@ export default Ember.ArrayController.extend(HasCurrentUser, {
         key: PreloadStore.get('stripe_key'),
         name: "Hummingbird PRO",
         description: this.get('selectedPlan.name'),
-        token: tokenCallback
+        token: tokenCallback,
+        amount: parseInt(this.get('selectedPlan.amount') * 100),
```
Comment (NuckChorris) [inline app/frontend/app/controllers/pro.js]: parseInt without a radix parameter of 10 isn't safe.  Specify it just to be safe.

Also shouldn't the multiplication be after it's turned into an integer, not before?

Comment (vevix) [inline app/frontend/app/controllers/pro.js]: Nah because then it would parse `3.99` to `3`.

Comment (NuckChorris) [inline app/frontend/app/controllers/pro.js]: Then `parseFloat`?

The parseInt is probably receiving a Number from the multiplication already and doing nothing except ensuring it's an integer (or converting Number -> String -> Number, I'm not sure how parseInt works internally).  If it's supposed to truncate, then `Math.floor` is probably more idiomatic?

Comment (vevix) [inline app/frontend/app/controllers/pro.js]: Just tested and it's not needed at all. `this.get('selectedPlan.amount') * 100` works, without issue.

Comment (NuckChorris) [inline app/frontend/app/controllers/pro.js]: :+1: Figured as much.  Dynamic types are magic like that :tophat: :sparkles:

**Comment by NuckChorris** [comment]
:moneybag: :+1: :sparkles: :money_with_wings: :slot_machine:

**Pull request merged**